### PR TITLE
Removed unnecessary aliasing of OneDimensionalArray #161

### LINKED ISSUE #161: Remove unnecessary aliasing of OneDimensionalArray
#### Description of the problem

See, https://github.com/codezonediitj/pydatastructs/blob/b0c3136dfaa6d8551adfa208bcdc76bb3f95dd4f/pydatastructs/linear_data_structures/arrays.py#L130

It has unnecessary aliasing of `OneDimensionalArray`. That line should be removed.

### PR DESCRIPTION
**Pull request by ShebinJoseph**
#### References to other Issues or PRs or Relevant literature

Fixes #161

### CODE REVIEW

**Comment by ShebinJoseph** [comment]
please review the pull request and do the needfull

**Comment by czgdp1807** [comment]
The tests are failing because there are multiple files where this aliasing was used. Places where `ODA` is written should be replaced with `OneDimensionalArray` along with correcting the imports in those files.

**Comment by ShebinJoseph** [comment]
all places where ODA is written should be interchanged to OneDimensionalArray,right ?

**Comment by ShebinJoseph** [comment]
I interchanged it,but when I runed it, I got a errors like
Instance of 'OneDimensionalArray' has no '_num' member
Instance of 'ArrayForTrees' has no '_load_factor' member
Access to member '_last_pos_filled' before its definition line 224
Whats the issue ?

**Comment by ShebinJoseph** [comment]
I changed it and commited again,but the fail is not going

File: pydatastructs/linear_data_structures/arrays.py [hunk 1]
```diff
@@ -48,8 +48,8 @@ class OneDimensionalArray(Array):
     Examples
     ========
 
-    >>> from pydatastructs import OneDimensionalArray as ODA
```
Comment (HarsheetKakar) [inline pydatastructs/linear_data_structures/arrays.py]: Maybe doctest is failing because u removed the import.

**Comment by HarsheetKakar** [comment]
remove trailing whitespaces and make changes in test_array.py

File: pydatastructs/linear_data_structures/arrays.py (hunk 1 above)
Comment (czgdp1807) [reply pydatastructs/linear_data_structures/arrays.py]: This needs to be restored.

File: pydatastructs/linear_data_structures/arrays.py [hunk 2]
```diff
@@ -48,8 +48,8 @@ class OneDimensionalArray(Array):
...
-    >>> arr = ODA(int, 5)
+
```
Comment (czgdp1807) [inline pydatastructs/linear_data_structures/arrays.py]:
```suggestion
    >>> from pydatastructs import OneDimensionalArray
```

File: pydatastructs/linear_data_structures/arrays.py [hunk 3]
```diff
@@ -1,4 +1,4 @@
-from pydatastructs.utils.misc_util import _check_type, NoneType
+from pydatastructs.utils.misc_util import NoneType, _check_type
```
Comment (czgdp1807) [inline pydatastructs/linear_data_structures/arrays.py]:
```suggestion
from pydatastructs.utils.misc_util import _check_type, NoneType
```

**Comment by czgdp1807** [comment]
Thanks.

**Pull request merged**